csrankings.csv

```diff
@@ -6703,6 +6703,7 @@ James J. Martin,Clemson University,https://people.cs.clemson.edu/~jmarty,66VE-3M
 James K. Baker,Carnegie Mellon University,http://ethw.org/James_K._Baker,9Z2rJjYAAAAJ
 James K. Hahn,George Washington University,https://www.seas.gwu.edu/~hahn,m0BU16YAAAAJ
 James Kwangjune Hahn,George Washington University,https://www.seas.gwu.edu/~hahn,m0BU16YAAAAJ
+James L. Patton,University of Illinois at Chicago,https://pattonj.people.uic.edu/,GWUT6MQAAAAJ

```

Comment:
Not in CS and no mention of an affiliation.

Reply:
Prof. Patton is listed on https://cs.uic.edu/faculty-staff/adjunct-faculty/ as adjunct faculty (can solely advise CS PhD students). Is it also required that he list this affiliation on his own page?

Reply:
Nope, that's fine - thanks!

Reply:
These diffs went stale so I rebased them - should be clean to merge now.